web/modules/custom/collection/config/install/core.base_field_override.rdf_entity.collection.label.yml

```diff
@@ -0,0 +1,17 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - rdf_entity.rdfentity.collection
+id: rdf_entity.collection.label
+field_name: label
+entity_type: rdf_entity
+bundle: collection
+label: 'Title'
+description: ''
+required: true
+translatable: true
+default_value: { }
+default_value_callback: ''
+settings: { }
+field_type: string
```

Comment:
Discussed this with @idimopoulos. If this base field is not overridden using this exported config, then the field will not be exposed in `hook_entity_bundle_field_info_alter()` and will not be alterable. So we need this.